### PR TITLE
Release 0.8.11.4.3

### PR DESCRIPTION
**Pull request by wuuuduu**

### CODE REVIEW

**Comment by mimi89999** [comment]
@wuuuduu This PR is about making a new release and adding testing for it. Please change the commit message and PR title.

File: tox.ini [hunk 1]
```diff
@@ -29,6 +29,7 @@ deps =
     dj200: Django<2.1
     dj201: Django<2.2
     dj202: Django<2.3
+    dj300: Django==3.0
```
Comment (mimi89999) [inline tox.ini]: Why did you change style? `dj300: Django<3.1` would look more appropriate.

File: CHANGES.md [hunk 2]
```diff
@@ -7,6 +7,9 @@
 
 Archived
 --------
+0.8.11.4.3 - Support for Django 3.0
+0.8.11.4.2 - Fix crispy-forms issue with SummernoteInplaceWidget
+0.8.11.4.1 - Security fixes and CustomUserPassesTestMixin in SummernoteUploadAttachment view
```
Comment (mimi89999) [inline CHANGES.md]: Unrelated entries in the changelog should probably be in a separate commit.

**Comment by leekyuchang** [comment]
How can I use this ?

**Comment by ulgens** [comment]
This PR adds Django 3.0 to matrix but the project doesn't work with 3.0.

**Comment by mimi89999** [comment]
@lqez Could you please make s new release adding support for Django 3.0.0?

**Comment by lqez** [comment]
Thanks. I just published `0.8.11.5` with Django 3.0 support.
https://github.com/summernote/django-summernote/releases
